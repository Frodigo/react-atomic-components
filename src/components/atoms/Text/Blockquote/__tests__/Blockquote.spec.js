import React from 'react';
import { render } from '@testing-library/react';

import BlockQuote from '../Blockquote';

describe('Blockquote should be rendered', () => {
    const quote = 'Lorem ipsum';
    const author = 'Marcin Kwiatkowski';

    test('Blockquote renders correctly', () => {
        const { container } = render(
            <BlockQuote quote={quote} author={author} />
        );

        expect(container).toMatchSnapshot();
    });

    test('Blockquote inverse renders correctly', () => {
        const { container } = render(
            <BlockQuote quote={quote} author={author} type="inverse" />
        );

        expect(container).toMatchSnapshot();
    });
});
