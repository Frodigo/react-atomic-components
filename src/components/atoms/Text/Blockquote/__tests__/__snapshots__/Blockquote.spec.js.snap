// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blockquote should be rendered Blockquote inverse renders correctly 1`] = `
<div>
  <figure
    class="figure"
  >
    <div
      class="inner"
    >
      <blockquote
        class="quote"
      >
        <span>
          svg
        </span>
        <p
          class="paraghaph"
          style="text-align: center;"
        >
          Lorem ipsum
        </p>
      </blockquote>
      <figcaption
        class="caption"
      >
        Marcin Kwiatkowski
      </figcaption>
    </div>
  </figure>
</div>
`;

exports[`Blockquote should be rendered Blockquote renders correctly 1`] = `
<div>
  <figure
    class="figure"
  >
    <div
      class="inner"
    >
      <blockquote
        class="quote"
      >
        <span>
          svg
        </span>
        <p
          class="paraghaph"
          style="text-align: center;"
        >
          Lorem ipsum
        </p>
      </blockquote>
      <figcaption
        class="caption"
      >
        Marcin Kwiatkowski
      </figcaption>
    </div>
  </figure>
</div>
`;
